src/aui-form-validator/tests/unit/js/tests.js

```diff
@@ -602,6 +602,160 @@ YUI.add('aui-form-validator-tests', function(Y) {
             Y.Assert.isFalse(input.hasClass('success-field'));
         },
 
+        /*
+         * @tests AUI-2820
+         */
+        'should add success-field and aria-invalid attr on all, named Nodes and their fieldContainer': function() {
```

Comment:
Also, this test description is confusing, is it supposed to say `on all named Nodes ...`, without the comma? If not, then `on all` what? Same for the next test.